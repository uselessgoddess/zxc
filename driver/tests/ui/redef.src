fn foo() {}
fn foo() {}
fn foo() {}
