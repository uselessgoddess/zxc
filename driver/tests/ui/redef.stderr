error:  `main` function not found in file
[#ERROR_PLACEHOLDER] Error: the name: `foo` is defined multiple times
   ╭─[redef:1:1]
   │
 1 │ fn foo() {}
 2 │ fn foo() {}
   │ ────┬───  
   │     ╰───── `foo` redefined here
───╯
[#ERROR_PLACEHOLDER] Error: the name: `foo` is defined multiple times
   ╭─[redef:1:1]
   │
 1 │ fn foo() {}
   │ 
 3 │ fn foo() {}
   │ ────┬───  
   │     ╰───── `foo` redefined here
───╯
